core/controllers/email_dashboard_test.py

```diff
@@ -294,9 +298,14 @@ def test_email_dashboard_result_page_with_invalid_query_id_raises_400(self):
             0)
 
         csrf_token = self.get_new_csrf_token()
-
+        params = {
+            'email_body': 'random_email_body',
+            'email_subject': 'random_email_subject',
+            'email_intent': 'random_email_intent',
```

Comment:
Can you please use the term valid instead of random for more clarity. Ditto throughout the file

Reply:
Replaced.